src/hakim/hakim.go

```diff
@@ -10,6 +10,8 @@ import (
 
 var requiredProcesses = []string{"consul.exe", "containerizer.exe", "garden-windows.exe", "rep.exe", "metron.exe"}
 
+var bbsCosulHost = "bbs2.service.cf.internal."
```

Comment:
What is this host? On our deployment, bbs is at `bbs.service.cf.internal`, not `bbs2`.